.github/workflows/c-cpp.yml

```diff
@@ -0,0 +1,23 @@
+name: C/C++ CI
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
```

Comment:
Definition from <a href="https://techterms.com/definition/cursor">The Tech Terms Dictionary</a>

Reply:
999b40769bec4006ad170db42cbeed8d91ce182e
 Is oke 
Treet